ChangeLog

```diff
@@ -54,6 +54,9 @@ API Changes
    * Extend the MBEDTLS_SSL_EXPORT_KEYS to export the handshake randbytes,
      and the used tls-prf.
    * Add public API for tls-prf function, according to requested enum.
+   * Add DER-encoded test CRTs to library/certs.c, allowing
```

Comment:
The two entries above are already in the Features section. I don't know which section they actually belong in though.

Reply:
This is pre-existing, and perhaps intended?